### ISSUE TITLE
error loading module 'libcuda_utils

### ISSUE THREAD

**Issue by fujinzhou** [luanfujun/deep-photo-styletransfer#56]
/home/hckj/torch/install/bin/luajit: /home/hckj/torch/install/share/lua/5.1/trepl/init.lua:389: error loading module 'libcuda_utils' from file './libcuda_utils.so':
        libTHC.so.0: cannot open shared object file: No such file or directory
stack traceback:
        [C]: in function 'error'
        /home/hckj/torch/install/share/lua/5.1/trepl/init.lua:389: in function 'require'
        neuralstyle_seg.lua:7: in main chunk
        [C]: in function 'dofile'
        ...hckj/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
        [C]: at 0x00406670

**Comment by fujinzhou** [comment]
Please help me see this question. thank

**Comment by mclarkestudio** [comment]
Same issue here. I have reinstalled torch.
Running this on mac.

matthewclarke/torch/install/share/lua/5.1/trepl/init.lua:389: error loading module 'libcuda_utils' from file './libcuda_utils.so':
	dlopen(./libcuda_utils.so, 6): no suitable image found.  Did find:
	./libcuda_utils.so: unknown file type, first eight bytes: 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00
stack traceback:
	[C]: in function 'error'
	...matthewclarke/torch/install/share/lua/5.1/trepl/init.lua:389: in function 'require'
	neuralstyle_seg.lua:7: in main chunk
	[C]: in function 'dofile'
	...arke/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x010bf65bb0

**Comment by stancil1** [comment]
Has anyone solved this problem?

**Comment by qq1227918543** [comment]
Problem?

**Comment by stancil1** [comment]
Having same issue where after I run the make file and try to run python gen_all.py I receive 

/usr/share/lua/5.3/trepl/init.lua:389: error loading module 'lubcuda_utils' from file './libcuda_utils.so':
stack traceback:
[C]: in function 'error'
/usr/share/lua/5.3/trepl/init.lua:389: in function 'require'
nerualstyle_seg.lua:7: in main chunk
[C]: in function 'dofile'
/usr/lib64/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
[C]: in ?

I think it may be an issue with torch trying to load libcuda_utils, but I"m not sure yet how to fix this problem.

**Comment by MarkOUzx** [comment]
I just delete it, and find it all right

**Comment by stancil1** [comment]
Delete what?

**Comment by stancil1** [comment]
So it looks like I have solved at least my current issue with this by having to edit the LUA_PATH and LUA_CPATH variables. Now it can find the correct packages.

**Comment by mpselland** [comment]
Yes set the variables to env_variables path for both 'LUA_PATH' and 'LUA_CPATH', it will probably find the path and correct the package missing issue.
